packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-container.jsx

```diff
@@ -81,6 +89,23 @@ const ProofOfAddressContainer = ({ is_mx_mlt, is_switching, has_restricted_mt5_a
         is_age_verified,
     } = authentication_status;
 
+    const from_platform = getPlatformRedirect(app_routing_history);
+
+    const should_show_redirect_btn = Object.keys(platforms).includes(from_platform.ref);
+
+    const redirect_button = should_show_redirect_btn && (
```

Comment:
`redirect_button` should return a button component.
You can move the condition to the prop level.
Also return `null` if the condition does not meet.

Reply:
Will refactor the code